imsv-docs-docusaurus/openapi/endpoints/login/models/signup-response.yaml

```diff
@@ -0,0 +1,60 @@
+type: object
```

Comment:
this has the same response as `login-init`. you can reuse it

Reply:
there's a slight different, just the mode value